syft/workers/base.py

```diff
@@ -336,15 +346,16 @@ def send_command(self, recipient, message):
 
         return response
 
-    def set_obj(self, obj):
+    def set_obj(self, obj: Union[torch.Tensor, AbstractTensor]) -> None:
```

Comment:
This is clever since all of the syft tensor types subclass abstract tensor. Good job